### DISCUSSION TITLE
Discrepancy between content of applicable codelist permission_einvoicing.gc for BT-743 and requirement expressed in rule BR-BT-00743-0100

### DISCUSSION THREAD

**Discussion by jlouvrier** [OP-TED/eForms-SDK#339] [Q&A] [answered]
Hi!
Rule BR-BT-00743-0100 expects the value "Yes" for BT-743. 

<entry key="rule|text|BR-BT-00743-0100">If Procedure Legal Basis (BT-01) value is different from 'Other', then Electronic Invoicing (BT-743) is equal to 'Yes'.</entry>

But "Yes" is not a value of the expected code list for this BT. Values in permission_einvoicing.gc: Allowed, not allowed, Required.

What do you think?

**Comment by YvesJo** [comment, marked as answer]
Hi,
Indeed the text message was created even before the codelist was created and should have been updated with the "required" code.
This should be fixed with the next release.
KR
